lib/protocol/frame.js

```diff
@@ -1,6 +1,12 @@
 /**
  * Change focus to another frame on the page. If the frame id is null,
  * the server should switch to the page's default content.
+ * <example>
+    :Using `element` to find an iframe and providing it to `frame`
```

Comment:
What should be the value of filename? I copied the format from the following link but I didn't understand what the filename stands for (couldn't find it anywhere). See https://github.com/webdriverio/webdriverio/edit/master/lib/commands/waitForExist.js